ops/storage.py

```diff
@@ -22,6 +22,10 @@
 import yaml
 
 
+def _run(args, **kw):
```

Comment:
Does it make sense to have a "multiplatform layer/module", instead of start adding small compatibility functions all around?

Also, docstring, thanks.

Reply:
are they really "all around" already? (if they were, i'd agree with you)

Reply:
Something to have in mind for the future, then!

Reply:
yeah, it's possible at some point the complexity of tweaks to make things work in both places is more than the complexity of tracking the evolution of two equivalent but subtly different code paths. We'll need to be alert to that.